css/styles.css

```diff
@@ -483,108 +488,225 @@ li {
 }
 
 @media (min-width: 1024px) {
-  .projects-section {
-    display: none;
-  }
-
-  .desktop-background {
-    display: block;
-    background-color: #e5e5e5;
-    padding: 12rem 4rem 12rem 4rem;
-  }
-
-  .desktop-project1,
-  .desktop-project2,
-  .desktop-project3,
-  .desktop-project4 {
-    background-color: #fff;
-    padding: 2rem;
-    border-radius: 2rem;
+  .project-display {
     display: flex;
-    gap: 3rem;
+    gap: 5rem;
     justify-content: center;
     align-items: center;
   }
 
-  .desktop-project1 {
-    justify-content: center !important;
+  .project-1--img {
+    width: 40%;
+    margin: 0;
   }
 
-  .desktop-project1 br {
-    display: none;
+  .project-1--heading {
+    margin-bottom: 2rem;
+    margin-top: 0;
   }
 
-  .desktop-project1-br1 {
-    display: block;
+  .project-1--paragraph {
+    margin-bottom: 2rem;
   }
 
-  .desktop-project2,
-  .desktop-project3,
-  .desktop-project4 {
-    margin-top: 10rem;
+  .project-1--paragraph2 {
+    margin-bottom: 1.5rem;
   }
 
-  .desktop-project2,
-  .desktop-project4 {
-    flex-direction: row-reverse;
+  .project1--paragraph4 {
+    margin-bottom: 1.5rem;
   }
 
-  .desktop-project2-br1,
-  .desktop-project2-br2 {
-    display: block;
+  .project-2:last-of-type {
+    margin-bottom: 3rem;
   }
+}
 
-  .project1-img1 {
-    width: 44rem;
-    height: 40rem;
-  }
+/* POP UP SECTION */
 
-  .text-tonic {
-    color: #091e42;
-    font-size: 4rem;
-    line-height: 5rem;
-    font-weight: 700;
-    margin-bottom: 2rem;
-  }
+.popup {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.7);
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+}
 
-  .text-display {
-    display: flex;
-    align-items: center;
-    gap: 2rem;
-    margin-bottom: 2rem;
-  }
+.popup-background {
+  position: fixed;
+  background-color: #fff !important;
+  width: 95%;
+  height: 95%;
+  left: 1rem;
+  right: 1rem;
+  top: 1rem;
+  bottom: 1rem;
+  padding: 1rem;
+  border-radius: 1rem;
+}
 
-  .text-canopy {
-    font-size: 1.8rem;
-    line-height: 2.4rem;
-    font-weight: 500;
-    color: #344563;
+.close-popup {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 2rem;
+}
+
+.popup-tonic {
+  font-size: 3rem;
+  color: #172b4d;
+  font-weight: 700;
+}
+
+.popup1-close-window {
+  margin-right: 3rem;
+  border: none;
+  background-color: transparent;
+  color: #67798e;
+  font-size: 1.3rem;
+  font-weight: 600;
+  transition: all 0.5s ease;
+}
+
+.popup1-close-window:hover {
+  font-size: 1.8rem;
+  cursor: pointer;
+}
+
+.popup-list1 {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  margin-bottom: 1rem;
+}
+
+.popup-canopy {
+  font-size: 1.3rem;
+  color: #344563;
+  font-weight: 600;
+}
+
+.popup-circle {
```

Comment:
- [ ] [OPTIONAL] Your popup circle on your modal doesn't show on the browser well. I kindly suggest that you fix this by adding a background color to your pop-up-circle 

| Template Design  | Your Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-20 19-08-09](https://user-images.githubusercontent.com/52934128/169590598-a8c2ec20-2a96-41ce-b2bd-b6ca1e8e196c.png)  | ![Screenshot from 2022-05-20 19-08-43](https://user-images.githubusercontent.com/52934128/169590355-0970cf53-0b2e-46d8-94e0-b132e3bcf1bd.png)  |
| ![Screenshot from 2022-05-20 19-08-23](https://user-images.githubusercontent.com/52934128/169590323-299f9fb4-b9d0-41b2-b6cf-e75e6d9a76a5.png)  | ![Screenshot from 2022-05-20 19-08-53](https://user-images.githubusercontent.com/52934128/169590378-a7590f53-ba19-4aba-86ec-653fba7243e5.png)  |